### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception Cannot read scheme com.intellij.configurationStore.SchemeManagerIprProvider: AZURE_FUNCTION_SUPPORT-Deploy Functions_ azure-function-examples_azure-function-examples_ factoryName_ Deploy Functions.xml com.intellij.util.xmlb.XmlSerializationException: Cannot deserialize class com.microsoft.azure.toolkit.intellij.legacy.function.runner.deploy.FunctionDeployModel$DeprecatedDeployModel

### ISSUE THREAD

**Issue by jcunningham672** [microsoft/azure-tools-for-java#6412]
IntelliJ build version: 2021.3.1 IU-213.6461.79
OS: Windows 11
JDK: JetBrains s.r.o. 11.0.13
Plugin version: 3.61.1-2021.3
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,8,0,1734x1012,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=Cannot read scheme com.intellij.configurationStore.SchemeManagerIprProvider: AZURE_FUNCTION_SUPPORT-Deploy Functions_ azure-function-examples_azure-function-examples_ factoryName_ Deploy Functions.xml, throwable=com.intellij.util.xmlb.XmlSerializationException: Cannot deserialize class com.microsoft.azure.toolkit.intellij.legacy.function.runner.deploy.FunctionDeployModel$DeprecatedDeployModel
	at com.intellij.util.xmlb.XmlSerializer.deserialize(XmlSerializer.java:58)
	at com.microsoft.azure.toolkit.intellij.legacy.function.runner.deploy.FunctionDeployConfiguration.lambda$readExternal$1(FunctionDeployConfiguration.java:172)
	at com.microsoft.azure.t

**Comment by Flanker32** [comment]
@jcunningham672 Thanks for your report, we will resolve this issue soon. As a workaround, could you please try create a new function run configuration for deploy?

**Comment by Flanker32** [comment]
@jcunningham672 Thanks for your report, the issue should be resolved in the latest release v3.62.0, could you please upgrade to the latest version and try again?
